x-pack/test/cases_api_integration/security_and_spaces/tests/trial/internal/user_actions_get_users.ts

```diff
@@ -0,0 +1,272 @@
+          headers: superUserHeaders,
+        });
+
+        // de-assignee secUser
+        await setAssignees({
+          supertest,
+          caseId: updatedCase[0].id,
+          version: updatedCase[0].version,
+          assignees: [{ uid: superUserProfile.uid }],
+          headers: superUserHeaders,
+        });
+
+        const { users } = await getCaseUsers({ caseId: postedCase.id, supertest });
+
+        expect(users).to.eql([
+          {
+            user: {
+              username: superUserProfile.user.username,
+              full_name: superUserProfile.user.full_name,
+              email: superUserProfile.user.email,
+              profile_uid: superUserProfile.uid,
+            },
+            type: 'participant',
+          },
+          {
+            user: {
+              username: secUserProfile.user.username,
+              full_name: secUserProfile.user.full_name,
+              email: secUserProfile.user.email,
+              profile_uid: secUserProfile.uid,
+            },
+            type: 'user',
+          },
+        ]);
+      });
+
+      it('does not return duplicate users', async () => {
+        const postedCase = await createCase(
+          supertestWithoutAuth,
+          getPostCaseRequest(),
+          200,
+          null,
+          superUserHeaders
+        );
+
+        // assignee superUser and secUserProfile
+        const updatedCase = await setAssignees({
+          supertest,
+          caseId: postedCase.id,
+          version: postedCase.version,
+          assignees: [{ uid: superUserProfile.uid }, { uid: secUserProfile.uid }],
+          headers: superUserHeaders,
+        });
+
+        // de-assignee secUser
+        const updatedCase2 = await setAssignees({
+          supertest,
+          caseId: updatedCase[0].id,
+          version: updatedCase[0].version,
+          assignees: [{ uid: superUserProfile.uid }],
+          headers: superUserHeaders,
+        });
+
+        // re-assign secUser
+        await setAssignees({
+          supertest,
+          caseId: updatedCase2[0].id,
+          version: updatedCase2[0].version,
+          assignees: [{ uid: superUserProfile.uid }, { uid: secUserProfile.uid }],
+          headers: superUserHeaders,
+        });
+
+        const { users } = await getCaseUsers({ caseId: postedCase.id, supertest });
+
+        expect(users).to.eql([
+          {
+            user: {
+              username: superUserProfile.user.username,
+              full_name: superUserProfile.user.full_name,
+              email: superUserProfile.user.email,
+              profile_uid: superUserProfile.uid,
+            },
+            type: 'participant',
+          },
+          {
+            user: {
+              username: secUserProfile.user.username,
+              full_name: secUserProfile.user.full_name,
+              email: secUserProfile.user.email,
+              profile_uid: secUserProfile.uid,
+            },
+            type: 'user',
```

Comment:
I think this user needs to be a participant since they were re-assigned to the case.